server/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+const EVENTS = {
```

Comment:
Maybe one for later but I'd consider using a typescript enum here instead of an object literal. The benefit of that would be, if `trackEvent` was called with anything other than a valid event name, the code won't even compile so the bug will be caught immediately.

Reply:
Fixed